test/e2e/asg-lifecycle-sqs-test

```diff
@@ -40,22 +40,6 @@ set -x
 helm "${localstack_helm_args[@]}"
 set +x
 
-aemm_helm_args=(
```

Comment:
We should fix the recent issue opened about the delay in starting up NTH if IMDS is not available because of the retires.  Removing this will slow down tests and might cause transient failures because of the assertion loop timeouts. I think it's the right move to remove AEMM from these test, just want to make sure the tests still pass consistently.

Reply:
on it like a bonnet 👒